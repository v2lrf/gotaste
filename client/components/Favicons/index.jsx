import React, { Fragment } from 'react'

function Favicons() {
  return (
    <Fragment>
      <link rel="shortcut icon" href="/static/gotaste.ico" />
      <link rel="icon" sizes="16x16 32x32 64x64" href="/static/gotaste.ico" />
      <link
        rel="icon"
        type="image/png"
        sizes="196x196"
        href="/static/gotaste-192.png"
      />
      <link
        rel="icon"
        type="image/png"
        sizes="160x160"
        href="/static/gotaste-160.png"
      />
      <link
        rel="icon"
        type="image/png"
        sizes="96x96"
        href="/static/gotaste-96.png"
      />
      <link
        rel="icon"
        type="image/png"
        sizes="64x64"
        href="/static/gotaste-64.png"
      />
      <link
        rel="icon"
        type="image/png"
        sizes="32x32"
        href="/static/gotaste-32.png"
      />
      <link
        rel="icon"
        type="image/png"
        sizes="16x16"
        href="/static/gotaste-16.png"
      />
      <link rel="apple-touch-icon" href="/static/gotaste-57.png" />
      <link
        rel="apple-touch-icon"
        sizes="114x114"
        href="/static/gotaste-114.png"
      />
      <link
        rel="apple-touch-icon"
        sizes="72x72"
        href="/static/gotaste-72.png"
      />
      <link
        rel="apple-touch-icon"
        sizes="144x144"
        href="/static/gotaste-144.png"
      />
      <link
        rel="apple-touch-icon"
        sizes="60x60"
        href="/static/gotaste-60.png"
      />
      <link
        rel="apple-touch-icon"
        sizes="120x120"
        href="/static/gotaste-120.png"
      />
      <link
        rel="apple-touch-icon"
        sizes="76x76"
        href="/static/gotaste-76.png"
      />
      <link
        rel="apple-touch-icon"
        sizes="152x152"
        href="/static/gotaste-152.png"
      />
      <link
        rel="apple-touch-icon"
        sizes="180x180"
        href="/static/gotaste-180.png"
      />
      <meta name="msapplication-TileColor" content="#FFFFFF" />
      <meta name="msapplication-TileImage" content="/static/gotaste-144.png" />
      <meta name="msapplication-config" content="/static/browserconfig.xml" />
    </Fragment>
  )
}

export default Favicons
