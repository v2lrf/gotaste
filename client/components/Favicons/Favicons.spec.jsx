import React from 'react'
import Favicons from '.'

describe('Favicons', () => {
  let component
  beforeEach(() => {
    component = render(<Favicons />)
  })

  describe('Snapshots', () => {
    it('should match the snapshot', () => {
      expect(component).toMatchSnapshot()
    })
  })
})
