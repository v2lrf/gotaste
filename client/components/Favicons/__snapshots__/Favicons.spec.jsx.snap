// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Favicons Snapshots should match the snapshot 1`] = `
Array [
  <link
    href="/static/gotaste.ico"
    rel="shortcut icon"
  />,
  <link
    href="/static/gotaste.ico"
    rel="icon"
    sizes="16x16 32x32 64x64"
  />,
  <link
    href="/static/gotaste-192.png"
    rel="icon"
    sizes="196x196"
    type="image/png"
  />,
  <link
    href="/static/gotaste-160.png"
    rel="icon"
    sizes="160x160"
    type="image/png"
  />,
  <link
    href="/static/gotaste-96.png"
    rel="icon"
    sizes="96x96"
    type="image/png"
  />,
  <link
    href="/static/gotaste-64.png"
    rel="icon"
    sizes="64x64"
    type="image/png"
  />,
  <link
    href="/static/gotaste-32.png"
    rel="icon"
    sizes="32x32"
    type="image/png"
  />,
  <link
    href="/static/gotaste-16.png"
    rel="icon"
    sizes="16x16"
    type="image/png"
  />,
  <link
    href="/static/gotaste-57.png"
    rel="apple-touch-icon"
  />,
  <link
    href="/static/gotaste-114.png"
    rel="apple-touch-icon"
    sizes="114x114"
  />,
  <link
    href="/static/gotaste-72.png"
    rel="apple-touch-icon"
    sizes="72x72"
  />,
  <link
    href="/static/gotaste-144.png"
    rel="apple-touch-icon"
    sizes="144x144"
  />,
  <link
    href="/static/gotaste-60.png"
    rel="apple-touch-icon"
    sizes="60x60"
  />,
  <link
    href="/static/gotaste-120.png"
    rel="apple-touch-icon"
    sizes="120x120"
  />,
  <link
    href="/static/gotaste-76.png"
    rel="apple-touch-icon"
    sizes="76x76"
  />,
  <link
    href="/static/gotaste-152.png"
    rel="apple-touch-icon"
    sizes="152x152"
  />,
  <link
    href="/static/gotaste-180.png"
    rel="apple-touch-icon"
    sizes="180x180"
  />,
  <meta
    content="#FFFFFF"
    name="msapplication-TileColor"
  />,
  <meta
    content="/static/gotaste-144.png"
    name="msapplication-TileImage"
  />,
  <meta
    content="/static/browserconfig.xml"
    name="msapplication-config"
  />,
]
`;
